tox.ini

```diff
@@ -28,6 +28,7 @@ commands = flake8 setup.py frontera
 [flake8]
 ignore = E265,E501,F401,W391,W292,E226
 exclude = frontera/_version.py,versioneer.py,docs/source/conf.py,frontera/contrib/backends/opic/discovery.py
+application-import-names = frontera, tests
```

Comment:
Hints for `flake8-import-order`. Needed to suppress invalid import order errors.